users/service_test.go

```diff
@@ -116,6 +116,49 @@ func TestLogin(t *testing.T) {
 }
 
 func TestUser(t *testing.T) {
+	svc := newService()
+	id, err := svc.Register(context.Background(), user)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s", err))
+
+	token, err := svc.Login(context.Background(), user)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s", err))
+
+	u := user
+	u.Password = ""
+
+	cases := map[string]struct {
+		user   users.User
+		token  string
+		userID string
+		err    error
+	}{
+		"view user with authorized token": {
+			user:   u,
+			token:  token,
+			userID: id,
+			err:    nil,
+		},
+		"view user with unauthorized token": {
+			user:   users.User{},
+			token:  "",
+			userID: id,
+			err:    users.ErrUnauthorizedAccess,
```

Comment:
why dont we put errors like this in separate package, `errors` maybe so that we can use it in each service not repeating it?